src/windows/Payload.Windows/Payload.Windows.csproj

```diff
@@ -28,8 +28,8 @@
       $(OutDir)GitHub.dll;
       $(OutDir)GitHub.UI.exe;
       $(OutDir)Microsoft.AzureRepos.dll;
-      $(OutDir)Microsoft.Git.CredentialManager.dll;
-      $(OutDir)Microsoft.Git.CredentialManager.UI.dll;">
```

Comment:
This is not a problem with the changes here, but looks like we've not been signing the `gcmcoreui.dll` assembly (to use the new names)! Oops..

Reply:
I believe this is fixed now!